bikeshed/include/status-webplatform-ED.include

```diff
@@ -0,0 +1,38 @@
+<p>
+  This is a public copy of the editors’ draft.
+  It is provided for discussion only and may change at any moment.
+  Its publication here does not imply endorsement of its contents by W3C.
+  Don’t cite this document other than as work in progress.
+
+<p>
+  <strong>Changes to this document may be tracked at
+  <a href="[REPOSITORYURL]">[REPOSITORYURL]</a>.</strong>
+
+<p>
+  The (<a href="[MAILINGLISTARCHIVES]">archived</a>) public mailing list
+  <a href="mailto:[MAILINGLIST]?Subject=%5B[SHORTNAME]%5D%20PUT%20SUBJECT%20HERE">[MAILINGLIST]</a>
+  (see <a href="http://www.w3.org/Mail/Request">instructions</a>)
+  is preferred for discussion of this specification.
+  When sending e-mail,
+  please put the text “[SHORTNAME]” in the subject,
+  preferably like this:
+  “[<!---->[SHORTNAME]] <em>…summary of comment…</em>”
+
+<p>
+  This document was produced by the
```

Comment:
It seems kind of silly that every working group will more or less have to end up with their own 5 or so copies of status.include files. So given that at this point this paragraph is pretty much the only thing that would actually differ between working groups (at least for the csswg, dap, fxtf, houdini and webappsec templates), would it make sense to have a separate "procuded-by.include" and have that vary for working groups, and just have generic status--ED.include etc files for the rest of the shared boilerplate?